frontend/pkg/frontend/frontend.go

```diff
@@ -572,7 +579,8 @@ func (f *Frontend) ArmResourceDelete(writer http.ResponseWriter, request *http.R
 	}
 	f.logger.Info(fmt.Sprintf("document deleted for resource %s", resourceID))
 
-	writer.WriteHeader(http.StatusAccepted)
+	// TODO: Eventually this will be an asynchronous delete and need to return a 202
+	writer.WriteHeader(http.StatusOK)
```

Comment:
Not sure this is really necessary.  We're never going to deploy to production with synchronous deletion.  We can leave it in, but I'm working on the async part of this now so it will be short-lived.

Reply:
100% short-lived, this is just for QE testing in P1. When they get to QE-ing async deletion, the different status code will be one part of their validation